build_resolvers/lib/src/resolver.dart

```diff
@@ -427,7 +427,9 @@ class AnalyzerResolvers implements Resolvers {
     _initAnalysisEngine();
     var resourceProvider = PhysicalResourceProvider.INSTANCE;
     var sdk = FolderBasedDartSdk(
-        resourceProvider, resourceProvider.getFolder(cli_util.getSdkPath()))
+        resourceProvider,
+        resourceProvider.getFolder(native_path
+            .dirname(native_path.dirname(Platform.resolvedExecutable))))
```

Comment:
oops, this one _is_ used internally and is broken by this... we have an internal only patch on `cli_util` that make it work in the build system...